### PR TITLE
fix: hide/unhide, full page scroll

### PR DESCRIPTION
**Pull request by amar-1995**
WEB-2907
WEB-2908
WEB-2910

### CODE REVIEW

**Comment by KaustubhKumar05** [comment]
image

Not able to scroll to the video section after the test. 

Since the show/hide details text is clickable, we can add cursor: pointer

**Comment by raviteja83** [comment]
@amar-1995 can we fix the scroll issue reported above.

File: packages/roomkit-react/src/Diagnostics/Diagnostics.tsx [hunk 1]
```diff
@@ -68,7 +84,7 @@ const DiagnosticsStep = () => {
   return (
     <Box css={{ border: '1px solid $border_default', r: '$1', w: '75%', maxWidth: '65rem', '@lg': { w: '100%' } }}>
       <DiagnosticsStepHeader />
-      <Box css={{ maxHeight: '55vh', overflowY: 'auto' }}>
+      <Box>
```
Comment (raviteja83) [inline packages/roomkit-react/src/Diagnostics/Diagnostics.tsx:87]: is Box needed?

File: packages/roomkit-react/src/Diagnostics/ConnectivityTest.tsx [hunk 2]
```diff
@@ -217,8 +239,6 @@ const ConnectivityTestReport = ({
   }
 
   if (result) {
-    // for debugging and quick view of results
-    console.log(result);
```
Comment (raviteja83) [inline packages/roomkit-react/src/Diagnostics/ConnectivityTest.tsx:221]: can be kept

**Review by raviteja83 — APPROVED** [review]

**Pull request merged**